@claude vercel deploy error

src/components/map/MapView.tsx(284,9): error TS2532: Object is possibly 'undefined'.
Error: Command "npm run build" exited with 2

--- a/src/components/map/MapView.tsx
+++ b/src/components/map/MapView.tsx
@@ -281,7 +281,8 @@ export default function MapView({
             <div style="font-size:11px;color:#6b7280;margin-top:1px">${escHtml(props.city_state ?? '')}</div>
           `)
           .addTo(m)
-        hoverPopupRef.current.getElement().style.pointerEvents = 'none'
+        const popupEl = hoverPopupRef.current.getElement()
+        if (popupEl) popupEl.style.pointerEvents = 'none'
       })
 
       m.on('mouseleave', 'unclustered-point', () => {

